src/main/msp/msp.c

```diff
@@ -2758,7 +2749,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         gpsRescueConfigMutable()->throttleMax = sbufReadU16(src);
         gpsRescueConfigMutable()->throttleHover = sbufReadU16(src);
         gpsRescueConfigMutable()->sanityChecks = sbufReadU8(src);
-        sbufReadU8(src);  // not used since 1.43, was gps rescue minSats
+        gpsRescueConfigMutable()->minSats = sbufReadU8(src);
```

Comment:
@klutvott123 Can we "revive" old deprecated values or do they need to be amended to the back of the list as "newly introduced in 1.45"?

Reply:
Normally it's illegal, but in this case, yes, and it's how it should be done here. If we were to add it to the end we would have to make configurator changes for nothing

Reply:
Yeah, didn't realize this was just resetting a recent change.